### PR TITLE
ceph-common: on rh storage on debian add the repositories and keys (ISO)

### PR DESCRIPTION
**Pull request by alfredodeza**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1337320

### CODE REVIEW

File: roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml [hunk 1]
```diff
@@ -1,4 +1,30 @@
 ---
+# NOTE: this assumes that the ISO will always have a MON directory. Current
+# ISOs have MON, OSD, Tools, and Agent. If ISOs change the layout or provide
+# certain repos depending on the type of node this task will require a more
+# intelligent way to determine the location of the key.
+- name: install the rh ceph storage repository key
+  apt_key:
+    data: "{{ ceph_stable_rh_storage_repository_path }}/MON/release.asc"
```
Comment (ktdreyer) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml]: nice

File: roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+    - "OSD"
+    - "Tools"
+    - "Agent"
+  when:
+    - ceph_stable_rh_storage
+    - ceph_stable_rh_storage_iso_install
+
 - name: add the red hat storage apt-key
   apt_key:
     url: "https://www.redhat.com/security/fd431d51.txt"
```
Comment (ktdreyer) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml]: Can we remove this, then?

Comment (ktdreyer) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml]: actually just conditionalize it - "when not ceph_stable_rh_storage_iso_install"

Comment (leseb) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml]: +1

Comment (alfredodeza) [inline roles/ceph-common/tasks/installs/install_rh_storage_on_debian.yml]: fixed in c78313d

**Pull request merged**